Sources/RInAppMessaging/Router.swift

```diff
@@ -137,4 +241,97 @@ internal class Router: RouterType {
             return rootView
         }
     }
+
+    private func findParentViewForTooltip(_ sourceView: UIView) -> UIView {
+        guard let superview = sourceView.superview else {
+            return sourceView
+        }
+
+        switch superview {
+        case is UIScrollView:
+            return superview
+        case is UIWindow:
+            if accessibilityCompatibleDisplay,
+               let transitionViewClass = NSClassFromString("UITransitionView"),
+               let transitionView = superview.subviews.first(where: { !$0.isKind(of: transitionViewClass) }) {
+
+                return transitionView
```

Comment:
It is actually correct - there's no space. It looks weird because the `let`s above are aligned to `if`. (The return statement has the same indentation as the one below)